Platform/ApollolakeBoardPkg/CfgData/Template_CfgData.dsc

```diff
@@ -54,27 +54,14 @@
 
     # !BSF NAME:{GPIO $(1) Data}
     # !HDR STRUCT:{PID_GPIO_PIN_DATA[]}
-    gCfgData.$(1)                       |      * | 0x04 | $(2)
+    gCfgData.$(1)                       |      * | 0x04 | 0x0000F0C4
```

Comment:
What is this magic value 0xF0C4 refer to?

Reply:
BSF HELP for this Combo Type explains this. LSByte is Community C4 (NORTHWEST) and 0x0F0 is Pad offset(8 * PinNumber). PMIC_STDBY GPIO Pin Number is 0x30 in that group